truffle/contracts/Splitter.sol

```diff
@@ -13,20 +15,22 @@ contract Splitter is Stoppable {
     
     function splitEth(address receiver1, address receiver2) public _onlyIfRunning payable returns(bool) {
```

Comment:
You can consider checking that the arguments have not been omitted. Otherwise they default to `0` -> burning Ether.

Reply:
You can name the return variable `returns(bool success)`.